src/sagemaker/serve/builder/model_builder.py

```diff
@@ -1256,8 +1258,10 @@ def _model_builder_optimize_wrapper(
             )
 
         if input_args:
+            print(input_args)
```

Comment:
is this supposed to be here? same for print below

Reply:
No, had it for debugging. I'll remove it.